projects/saturn/src/main/java/io/fairspace/saturn/services/metadata/MetadataService.java

```diff
@@ -245,7 +244,9 @@ public void patch(Model model) {
                                     resource.getURI(),
                                     label,
                                     e.getMessage());
-                            throw new BadRequestException(message);
+                            var violations = new LinkedHashSet<Violation>();
+                            violations.add(new Violation(message, resource.getURI(), "Label", label));
+                            throw new ValidationException(violations);
```

Comment:
ValidationException doesn't inherit from runtime-exception, so its a checked exception. Should the method than not declare it throws a ValidationException?

Reply:
actually it's a runtime exception (see [ValidationException.java](https://github.com/thehyve/sils-fairspace/blob/FDS-52-validation-error/projects/saturn/src/main/java/io/fairspace/saturn/services/metadata/validation/ValidationException.java))